Shuttle-back/src/main/java/com/shuttle/common/CollectionDTO.java

```diff
@@ -0,0 +1,22 @@
+package com.shuttle.common;
+
+import java.util.Collection;
+
+public class CollectionDTO<T> {
```

Comment:
This is clever, but I think we should specialize this class to use `List<T>` just in case, since the JSON schema expects a list.

Reply:
I agree